services/web/server/src/simcore_service_webserver/catalog/_catalog_rest_client_service.py

```diff
@@ -29,6 +32,16 @@
 
 _logger = logging.getLogger(__name__)
 
+# Cache settings
+_SECOND = 1  # in seconds
+_MINUTE = 60 * _SECOND
+_CACHE_TTL: Final = 1 * _MINUTE
```

Comment:
In my mind there is nothing against making this either
- longer (5 minutes?)
- configurable via an env-var ;) Then we can play on master and check for the sweet spot... 

;)